src/validation.cpp

```diff
@@ -5288,3 +5288,10 @@ std::optional<int> ChainstateManager::GetSnapshotBaseHeight() const
     auto base{this->GetSnapshotBaseBlock()};
     return base ? std::make_optional(base->nHeight) : std::nullopt;
 }
+
+bool ChainstateManager::IsAnyChainInIBD()
```

Comment:
Seems weird not to take the lock in the function, rather than have every caller invoke it as `WITH_LOCK(cs_main, ...)`